### ISSUE TITLE
Null pointer exception for danRer10 genome during viewpoint creation (Simple, HindIII) for all protein coding genes

### ISSUE THREAD

**Issue by hansenp** [TheJacksonLaboratory/Gopher#367]
When I try to create simple viewpoints for all protein coding genes of danRer10 using the restriction enzyme HindIII, I get a `NullPointerException` for the gene `cdkal1`.

![image](https://user-images.githubusercontent.com/10495485/61626297-3add9380-ac7d-11e9-8625-dc945001fa90.png)

**Comment by hansenp** [comment]
The TSS of `cdkal1` is on the first restriction digest of `chr16`.

![image](https://user-images.githubusercontent.com/10495485/61626579-e71f7a00-ac7d-11e9-8e47-a5dff2ffea0c.png)

**Comment by hansenp** [comment]
The exception does not occur for the extended approach.

**Comment by hansenp** [comment]
GOPHER tries to fetch a detailed report about the digest upstream of the center (TSS) digest even though it does exist. 
 
```
                if (genomicPosFragIdx > 0) {
                    upstreamSegment = restrictionSegmentList.get(genomicPosFragIdx - 1);
                    logger.trace("SELECTED UPSTREAM " + upstreamSegment.detailedReport() );
                } else {
                    logger.trace("Warning: There is no segment in upstream direction of the center segment!");
                }
                if (genomicPosFragIdx < restrictionSegmentList.size() - 1) {
                    downstreamSegment = restrictionSegmentList.get(genomicPosFragIdx + 1);
                    logger.trace("SELECTED DOWNSTREAM " + upstreamSegment.detailedReport());  // <= Bug! Should be: upstreamSegment.detailedReport().

                } else {
                    logger.trace("Warning: There is no segment in downstream direction of the center segment!");
                }
```
I changed
`
logger.trace("SELECTED DOWNSTREAM " + upstreamSegment.detailedReport());`
to
`logger.trace("SELECTED DOWNSTREAM " + downstreamSegment.detailedReport());`

After this, viewpoints could be created without exception for all protein coding genes.

**Comment by hansenp** [comment]
Fixed bug and released version v0.6.1.
